### PR TITLE
[Fix] using datetime.datetime objects for all cases

### PR DESCRIPTION
**Pull request by intero-chz**
Fixing an exception when datetime like string is in context

### CODE REVIEW

File: currency_rate_date_check/currency_rate_date_check.py [hunk 1]
```diff
@@ -73,11 +73,8 @@ def _get_current_rate(
                         (currency_id, date))
                     if cr.rowcount:
                         rate_date = cr.fetchone()[1]
-                        rate_date_dt = fields.Date.from_string(rate_date)
-                        if len(date) <= 10:
-                            date_dt = fields.Date.from_string(date)
-                        else:
-                            date_dt = fields.Datetime.from_string(date)
+                        rate_date_dt = fields.Datetime.from_string(rate_date)
```
Comment (pedrobaeza) [inline currency_rate_date_check/currency_rate_date_check.py:76]: Isn't this going to raise an error of not enough data to read?

Comment (intero-chz) [inline currency_rate_date_check/currency_rate_date_check.py:76]: If i don't understand https://github.com/odoo/odoo/blob/8.0/openerp/fields.py#L1256 wrong, no.

Or what exactly did you mean?

Comment (pedrobaeza) [inline currency_rate_date_check/currency_rate_date_check.py:76]: OK, that's exactly what I mean. Thanks for the explanation.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by gurneyalex** [comment]
@intero-chz are your related to Intero Technologies GmbH?

**Comment by intero-chz** [comment]
Yes, i'm related.

**Comment by adrienpeiffer** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Pull request merged**